src/isAnyTrust.unit.test.ts

```diff
@@ -16,7 +17,20 @@ it('should return true for AnyTrust chain', async () => {
   expect(isPlaynanceAnyTrust).toBeTruthy();
 });
 
-it('should return false for non AnyTrust chain', async () => {
+it('should return true for AnyTrust chain (RollupCreator v2.1)', async () => {
+  const client = createPublicClient({
+    chain: arbitrumSepolia,
+    transport: http(),
+  });
+  // https://sepolia.arbiscan.io/tx/0xc1d9513cee57252ab9a0987e3ac4bf23aca7f5c58478a29439ecb1ef815cd379
```

Comment:
this chain's deploy param has lots of batch posters which is great for testing

i don't see this tx used for any other tests in the repo for the new batch posters [], perhaps you can use it on other tests?

Reply:
added to Linear